parm/atm/variational/3dvar_dripcg.yaml

```diff
@@ -33,7 +33,7 @@ variational:
   minimizer:
     algorithm: DRIPCG
   iterations:
-  - ninner: 2
```

Comment:
we should revert the changes to this file, it shouldn't change in the PR.

Reply:
@CoryMartin-NOAA I accidentally commited this in.  Try a few things but I could not revert this file to the base (feature/gdas-validation) successfully.  Could you help?

Reply:
@emilyhcliu I reverted, FYI I used:
`git checkout feature/gdas-validation -- parm/atm/variational/3dvar_dripcg.yaml`

Reply:
Supre!! Thank you @CoryMartin-NOAA